### DISCUSSION TITLE
Nature reserves

### DISCUSSION THREAD

**Discussion by mattlucht** [digital-land/data-standards-backlog#48] [Planning consideration]
Local and national nature reserves have been identified as a planning consideration that will add value to the PlanX service. The purpose of this discussion is to ask any questions that come up during the data modeling of nature reserves and to record any design decisions that are made.

**Comment by colmjude** [comment]
## What we've looking for

We want to know more about:

* what nature reserves are used for?
* what do nature reserves mean to you?
* what decisions are made using nature reserves?
* if you use nature reserve data now, how do you use it?
* what do you think is missing from the data?

**Comment by colmjude** [comment]
## Draft schemas

We have drafted schemas for [National nature reserves](https://github.com/digital-land/specification/blob/main/content/dataset/national-nature-reserve.md) and [Local nature reserves](https://github.com/digital-land/specification/blob/main/content/dataset/local-nature-reserve.md). We decided to keep these things separate for now - in the future we might join them into one `nature reserves` dataset. However, data already exists for the two things and is published separately, so we decided to keep them separate for now.

Looking at the available data, it maps nicely onto fields in our house style.  The schemas for both contain these fields

```
reference - an id for the reserve
name - the name of the reserve
organisation - the organisation that published the data
geometry - the wkt for the geospatial area
entry-date - the date the record was created
start-date - the date the reserve was designated
end-date - the date, if any, the area is no longer classified as a reserve
```

We have 2 outstanding questions about the existing data:
* every reserve has a `status` field and the value is always `declared`. What does this mean? Can it ever be anything else?
* there is a `GIS_DATE` - is this the date the reserve was designated or the date it was added to GIS?

**Comment by mattlucht** [comment]
Nature reserves, both local and national have been identified by ODP as important datasets to support householder applications in PlanX. By making this data available for PlanX to use in their service it'll mean that it's one less question (or one less caveat) that'll need to be applied within that service.
